src/modules/commander/commander_params.c

```diff
@@ -143,11 +146,13 @@ PARAM_DEFINE_INT32(COM_HLDL_REG_T, 0);
 PARAM_DEFINE_FLOAT(COM_RC_LOSS_T, 0.5f);
 
 /**
- * Delay between RC loss and configured reaction
+ * Delay between RC loss and configured action
  *
- * RC signal not updated -> still use data for COM_RC_LOSS_T seconds
- * Consider RC signal lost -> wait COM_RCL_ACT_T seconds in Hold mode to regain signal
- * React with failsafe action NAV_RCL_ACT
+ * If the RC signal does not update for COM_RC_LOSS_T seconds the RC connection is considered lost
+ * (during this interval data from the last RC update is still used).
+ * If the connection is lost, PX4 will enter Hold mode and wait for COM_RCL_ACT_T seconds
+ * to recover the connection before triggering the failsafe action (NAV_RCL_ACT).
+ * If the signal is regained PX4 will return to the previous mode.
```

Comment:
```suggestion
 * If the signal is regained during this timeout PX4 will return to the previous mode.
```
IMHO it should be but it's not currently on main.